README.md

```diff
@@ -5,19 +5,42 @@
 
 CleanVision automatically detects various issues in image datasets, such as images that are: (near) duplicates,
 blurry, over/under-exposed, etc. This data-centric AI package is designed as a quick first step for any computer vision
-project to find problems in your dataset, which you may want to address before applying machine learning.
+project to find problems in your dataset, which you may want to address before applying machine learning. It currently detects issues in the raw images themselves, and can be useful for any machine learning task (classification, segmentation, object detection, pose estimation, keypoint detection, image generation, etc).
+The list of all issues detected by CleanVision can be found in the table below.
+
+
+|   | Issue Type       | Description                                               | Issue Key        | Example                                      |
+|---|------------------|-----------------------------------------------------------|------------------|----------------------------------------------|
+| 1 | Dark             | Irregularly dark images                                   | dark             | ![](docs/readme_images/dark.jpg)             |
+| 2 | Blurry           | Blurry or out of focus images                             | blurry           | ![](docs/readme_images/blurry.png)           |
+| 3 | Grayscale        | Images lacking color                                      | grayscale        | ![](docs/readme_images/grayscale.png)        |
```

Comment:
I would only show the left image out of these two, is there a reason to show 2 images for "grayscale"? 

Also if possible, I'd choose an image that is more obviously black&white rather than these images where it isn't 100% obvious.

<img width="251" alt="Screen Shot 2023-03-09 at 10 52 42 PM" src="https://user-images.githubusercontent.com/1390638/224244696-08fc1bc7-eb35-41a1-9080-c0720e25c50f.png">